schemas/answers/text_field.json

```diff
@@ -26,6 +26,9 @@
       "mandatory": {
         "type": "boolean"
       },
+      "visible": {
```

Comment:
Should also be added to numbers.json as details answer can be a text field or a number.

Reply:
Added, thanks.